File: src/BackEnd/autos_sanchez_Back/src/main/java/net/autossanchez/filter/FilterUser.java
package net.autossanchez.filter;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import net.autossanchez.entity.Municipality;
import net.autossanchez.entity.Provinces;
import net.autossanchez.entity.User;
import net.autossanchez.entity.Vehicle;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class FilterUser {
	private User user;
	private Vehicle vehicle;
	private Municipality municipality;
	private Provinces province;
	private boolean codeStatus;
}

File: src/FrontEnd/src/app/interfaces/filter-user.ts
import { Municipality } from '../model/Municipality';
import { Provinces } from '../model/Provinces';
import { User } from '../model/User';
import { Vehicle } from '../model/Vehicle';

export interface FilterUser {  
    user?:User;
    vehicle?:Vehicle;
    municipality?:Municipality;
    province?:Provinces;
    codeStatus?:boolean;
}
